### PR TITLE
4716 deny access to draft or refused titles if no edit rights

### PR DESCRIPTION
**Pull request by RemcoSimonides**
Fix issue #4716 

Users are automatically redirected to homepage if they try to access the title page. This is because they can't read the movie doc due to the rules and therefore they can't pass the guard

### CODE REVIEW

**Review by phpgeekfr — COMMENTED** [review]
LGTM but can you add more test cases ?

- user can read movie in draft if in good org ?
- any user can read accepted movie ?

**Review by phpgeekfr — APPROVED** [review]
LGTM, maybe @xmano can have a look also

**Review by xmano — APPROVED** [review]
LGTM

File: apps/rules/src/tests/firestore/movies.spec.ts [hunk 1]
```diff
@@ -29,11 +30,16 @@ describe('Movies Rules Tests', () => {
     afterAll(() => Promise.all(apps().map((app) => app.delete())));
 
     describe('Read Movie', () => {
-      test('should be able to read movie', async () => {
-        const movieRef = db.doc('movies/M001');
+      test('should be able to read movie with status draft', async () => {
```
[the remark this answers is not in the document]
Comment (RemcoSimonides) [reply apps/rules/src/tests/firestore/movies.spec.ts:33]: This user is able to edit the movie and therefore is allowed to read it

**Pull request merged**